### ISSUE TITLE
NoMethodError in Admin::EventsController#comment

### ISSUE THREAD

**Issue by sonalkr132** [openSUSE/osem#888]
Steps to reproduce:
* Make a conference (say `goto`), assign cfp dates
* Visit home page and submit a proposal to `goto`
* Visit `events` show page through admin area (url: http://localhost:3000/admin/conference/goto/program/events/1`)
* Click on `comments (0)`, a textarea will appear.
* Fill in your comment and click `Add comment`

You will get:
> NoMethodError in Admin::EventsController#comment
undefined method `permit' for "sdfsdfsd":String

pointing to following [line](https://github.com/openSUSE/osem/blob/813658fb196cd00c2a95da419e68158d663b1343/app/controllers/admin/events_controller.rb#L179):
```Ruby
    def comment_params
        params.require(:comment).permit(:commentable, :body, :user_id)  # <---
    end
```

**Comment by differentreality** [comment]
Closed in #894

**Comment by Rolence** [comment]
it works.
![fixed_comment_view](https://cloud.githubusercontent.com/assets/11888675/13980636/fef56af2-f0df-11e5-9a56-fbb0e45916bf.png)
